.circleci/bin/test-airflow-image.py

```diff
@@ -316,6 +316,44 @@ def test_labels_for_onbuild_image(docker_client):
         "'maintainer' label should be 'Astronomer <humans@astronomer.io>'"
 
 
+@pytest.mark.skipif(not is_edge_build, reason="Not needed for non-Edge/main builds")
+def test_labels_for_edge_builds(docker_client):
+    # Note that this list does not include
+    # org.apache.airflow.git.branch or org.apache.airflow.git.tag
+    # because those aren't guarateed to tbe in the labels.
+    # They are checked seprately down below.
+    labels_to_check = [
+        'org.apache.airflow.ci.build.date',
+        'org.apache.airflow.ci.build.url',
+        'org.apache.airflow.ci.build.version',
+        'org.apache.airflow.ci.js.node.version_string',
+        'org.apache.airflow.ci.js.npm.version_string',
+        'org.apache.airflow.ci.js.yarn.version_string',
+        'org.apache.airflow.ci.python.version_string',
+        'org.apache.airflow.git.commit_sha',
+        'io.astronomer.astronomer_certified.build.version',
+    ]
+
+    image_labels = get_labels(docker_client)
+    for label in labels_to_check:
+        assert label in image_labels,
+            (f"'{label}' should be in image labels for edge build (image labels: "
+             f"{','.join(image_labels.keys())})")
```

Comment:
This failed actually:

```    return compile(source, filename, mode, flags,
E     File "/home/circleci/project/.circleci/bin/test-airflow-image.py", line 339
E       assert label in image_labels,
E                                   ^
E   SyntaxError: invalid syntax

```

Reply:
Should be fixed in https://github.com/astronomer/ap-airflow/pull/357/commits/88be37a261457b60882f568162b3182d87a1d951

Reply:
Thanks!